docs/docker.rst

```diff
@@ -22,7 +22,7 @@ On Ubuntu:
 
 * If you have difficulties with the ``apt`` install, try the ``pip`` install.
 * The docker group is automatically created. `Adding your user to the docker
-  group <https://docs.docker.com/v1.8/installation/ubuntulinux/#create-a-docker-group>`_
+  group <https://askubuntu.com/questions/477551/how-can-i-use-docker-without-sudo>`_
```

Comment:
This is a better link: https://docs.docker.com/engine/installation/linux/linux-postinstall/#manage-docker-as-a-non-root-user